Application/EdFi.Ods.WebApi/appsettings.json

```diff
@@ -12,6 +12,7 @@
     },
     "Tenants": {
     },
+    "StartUpClassName": "Startup",
```

Comment:
Rename this setting to "StartupClassName", and default it to an empty string (which should cause the existing `Startup` class to be used by default, as described above). This also reduces the chance of this change breaking someone who inadvertently misses the setting in their configuration.